api/config/sync-catalog/32-sync-catalog-cronjob.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: catalog-refresh
+  namespace: tekton-hub
+  labels:
+    app: api
+spec:
+  schedule: "*/30 * * * *"
```

Comment:
we need to maintain only the last 3/5 jobs, rest can be garbage collected like we do now in `experimental`